lizrd/core/llm.py

```diff
@@ -306,6 +365,11 @@ def __init__(
         device: torch.device = None,
         model_fragmentation: Optional[list[int]] = None,
         residual_fn: Optional[Callable] = None,
+        rank=None,
+        wrap_blocks_in_fsdp=False,
+        wrap_attn_and_ff_in_fsdp=False,
+        param_precision=torch.float32,
+        offload_params=False,
```

Comment:
rename to `fsdp_cpu_offloading`

Reply:
`cpu_offload_in_fsdp`

Reply:
I like signalling that an argument is strictly tied to`fsdp` on prefix. What do you think?

Reply:
ok